test/fixtures/es-class-assign-property-variable/expected-wrap-es5.js

```diff
@@ -1,8 +1,8 @@
 "use strict";
 
-var propTypes = {
+var propTypes = process.env.NODE_ENV !== "production" ? {
   foo: PropTypes.string
-};
+} : {};;
```

Comment:
are the double semicolons intended?

Reply:
> There seems to be a weird issue where the transform adds a semicolon
> when using in wrapped mode. For now, I decided to simply add the
> additional semicolons in the test fixtures, since they will be minified
> out anyway.

If you know why they are there, I'd love to fix!